cyhy_report/customer/report.mustache

```diff
@@ -755,6 +759,73 @@ As you review the report, you may have additional questions. Check out the answe
   \end{tikzpicture}
 \restoregeometry
 
+<<#owner_is_federal_executive>>
+\section{Binding Operational Directive 22-01 --- Reducing the Significant Risk of Known Exploited Vulnerabilities}<</owner_is_federal_executive>><<^owner_is_federal_executive>>\section{Reducing the Significant Risk of Known Exploited Vulnerabilities}<</owner_is_federal_executive>>
+\label{sec:known-exploited-vulnerabilities}
+
+Malicious cyber campaigns frequently use \glspl{KEV} to threaten the public
+sector, the private sector, and ultimately the security and privacy of
+individual citizens. Therefore it is essential to quickly remediate \glspl{KEV}
+to protect
+<<#owner_is_federal_executive>>federal<</owner_is_federal_executive>><<^owner_is_federal_executive>>your<</owner_is_federal_executive>>
+information systems and reduce cyber incidents<<^owner_is_federal_executive>>
+affecting critical infrastructure<</owner_is_federal_executive>>.
```

Comment:
I just want to make sure I'm reading this correctly that the `affecting critical infrastructure` is for non-federal/executive only?

Reply:
Yes, you are reading that correctly.  That is how it was written up in the mockups we were given.

Fed Executive: 
> Therefore it is essential to quickly remediate KEVs to protect federal information systems and reduce cyber incidents.

Non-Fed Executive:
> Therefore it is essential to quickly remediate KEVs to protect your information systems and reduce cyber incidents affecting critical infrastructure.